### PR TITLE
prow: cleanup archived repos

### PR DESCRIPTION
**Pull request by tuminoid**
Remove archived repos from prow config.

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: prow/config/config.yaml [hunk 1]
```diff
@@ -35,20 +35,16 @@ tide:
   queries:
     - repos:
         - metal3-io/baremetal-operator
-        - metal3-io/base-image
         - metal3-io/cluster-api-provider-metal3
         - metal3-io/ironic-client
         - metal3-io/ironic-hardware-inventory-recorder-image
```
Comment (furkatgofurov7) [inline prow/config/config.yaml:40]: how about this repo?

File: prow/config/config.yaml [hunk 2]
```diff
@@ -644,7 +640,7 @@ presubmits:
     - name: build
       branches:
         - main
-      run_if_changed: '^api|^test|^Makefile$'
+      run_if_changed: "^api|^test|^Makefile$"
```
Comment (furkatgofurov7) [inline prow/config/config.yaml:643]: single quotes have been used consistently in the file afaik, so please keep it aligned

**Review by furkatgofurov7 — APPROVED** [review]

File: prow/config/config.yaml (hunk 2 above)
Comment (furkatgofurov7) [reply prow/config/config.yaml:643]: /hold

to resolve ^ comment

File: prow/config/config.yaml (hunk 1 above)
Comment (tuminoid) [reply prow/config/config.yaml:40]: I checked it and it has not been archived. I does not really have any other prow config etc, but its not read-only like others I'm removing.

File: prow/config/config.yaml (hunk 2 above)
Comment (tuminoid) [reply prow/config/config.yaml:643]: This is weird editor glitch. It changes this quote only when auto-formatting, but doesn't touch any of the other quotes. I think Kashif's prow config change has same thing. Well, fixed now.

**Review by furkatgofurov7 — COMMENTED** [review]
Thanks!

/lgtm

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Pull request merged**